.github/workflows/status-embed.yaml

```diff
@@ -7,24 +7,12 @@ on:
     types:
       - completed
 
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
```

Comment:
What does this do and why?

Reply:
It means that if a status-embed workflow is triggered while one is already running it will kill the old one before running the new one. So if you quickly but separately push multiple commits in sequence it will only post the status of the newest CI run*.

I mainly added for consistency as most other repos had it already.

*now that I think about it, it's a bit more complicated than this, since the CI workflow takes longer than the status_embed workflow. If CI is replaced with a new run, the old one's status embed (saying it was cancelled) will finish before the new CI finishes, so both status embeds will be sent. However if the CI is rerun again, then it might cancel the old status embed and send the two newest... The behaviour is somewhat questionable but I don't think it's too bad...

Reply:
Thanks. If we already use it successfully in other repos then I'm not concerned with keeping it.